style.css

```diff
@@ -0,0 +1,29 @@
+.items li {
+    cursor: pointer;
+}
+
+li.todo:hover {
```

Comment:
TODO change slightly so that its different from li.done because its confusing when you unfinish a task and the strike through is still there.